src/automata/fsm.cljc

```diff
@@ -21,17 +21,18 @@
 
    Both arities throw when no possible transition was found"
   ([{::keys [rules state] :as machine} event]
-   (let [{::keys [actions to] :as transition} (transit rules state event)]
+   (let [{::keys [actions to] :as _transition} (transit rules state event)]
```

Comment:
seems as though `_transition` is never used, might as well remove it

Reply:
Yes, that's why I put the underscore, I tought you put it for doc purposes. 
Apparently joker follows erlang conventions for this and ignores it when prefixed with _ that way.